### DISCUSSION TITLE
How can get all non-inertial links in js?

### DISCUSSION THREAD

**Discussion by 313ava** [inertiajs/inertia#2128] [Help]
Hi, is there a way to distinguish between inertia and non-inertia links in js? I want get all non-inetia links but can't find any solution.

**Comment by RobertBoes** [comment]
What exactly are you trying to do? What do you mean with "get all non-inertia links"?

**Reply by 313ava** [reply]
inertia links have been generated with Link tags. but i have some dynamic content which contains some a tags. I want get all a tags and handle their click action with inertia manual visit.
